Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1452,9 +1452,9 @@ namespace Babylon
                 env.Global().Set(JS_CLASS_NAME, func);
             }
 
-            static Napi::Object New(const Napi::CallbackInfo& info)
+            static Napi::Object New(const Napi::Env& env)
```

Comment:
nit: I'm not sure I ever fully understood the reason, but I seem to remember that the best practice for `Napi::Env` is to pass it by copy. @bghgary, is this still true (if it ever was), and if so, do you remember the reason?

Reply:
Env is intended to be a thin wrapper around the underlying C pointer. Passing by ref does nothing in this case. I don't think there is a big difference though.

Reply:
would folks prefer if i change this here or should the reference suffice?

Reply:
ill just go a head and change this for now